src/flavors/raingardens/config.yml

```diff
@@ -158,6 +159,36 @@ place_types:
           iconAnchor: [25, 60]
           shadowSize: [50, 60]
           shadowAnchor: [14, 60]
+  idea:
+    label: _(idea)
```

Comment:
Hey Luke, I looked into this, and the caps are the way they should be.

In both Hey Duwamish and Raingardens configs, the location_type parameter is set as a lowercase because it avoids typos from having to remember to hold shift and capitalize it

However...as far as the place label that shows up in the app itself, in certain cases the label should be capitalized and in certain cases it should be lowercase.

In Hey Duwamish it's capitalized Idea, Observation, Question, etc. because it's in the Sidebar Legend and looks better capitalized.

In Raingardens, it looks better lowercase because otherwise in the syntax it would show up as "George added this Raingarden"  or  "George added this Idea"  rather than "George added this raingarden..."

I think it should be lowercase because it looks a little weird capitalized in the middle of a sentence.